src/translate_c/ast.zig

```diff
@@ -0,0 +1,267 @@
+const std = @import("std");
+const Type = @import("../type.zig").Type;
+
+pub const Node = struct {
```

Comment:
Looks like this is meant to be a union, yeah?

Reply:
Yup, haven't yet gotten this to compile.